src/material-experimental/mdc-button/icon-button.scss

```diff
@@ -26,4 +27,8 @@
       opacity: 1;
     }
   }
+
+  @include a11y.high-contrast(active, off) {
```

Comment:
The non-icon button also increases the outline to 3px on focus. Should we have the same here? It might be a good idea to pull these styles into `_button-base.scss` so they can be reaused.